django_test_migrations/autonames.py

```diff
@@ -31,8 +37,16 @@ def check_migration_names(*args, **kwargs) -> List[CheckMessage]:
         settings, _SETTINGS_NAME, set(),
     )
 
+    ignored_apps: FrozenSet[str] = frozenset(
+        migration[0] for migration in ignored_migrations
+        if migration[1] == _IGNORE_APP_MIGRATIONS_SPECIAL_KEY
```

Comment:
This generator expression is fine, however we can make it even more readable:

```suggestion
        app_label for app_label, migration_name in ignored_migrations
        if migration_name == _IGNORE_APP_MIGRATIONS_SPECIAL_KEY
```

Reply:
yep, looks great